### ISSUE TITLE
Terrible result during training

### ISSUE THREAD

**Issue by liuyangwen** [xingyizhou/pytorch-pose-hg-3d#49]
Hi Xingyi,
When I was training in stage2&3, I found that the accuracy and MPJPE is so terrible. I noticed accuracy drop from 0.83 to 0.02  in the first epoch of stage1! Is that possible reason for such case? 

Here is the log. pytorch-gpu version=0.3.1
![image](https://user-images.githubusercontent.com/11901126/50262016-14871980-044a-11e9-9d3a-69520b1b8035.png)

**Comment by xingyizhou** [comment]
Hi, 
  Thanks for reporting. This is a known issue of the pytorch cudnn BN implementation https://github.com/xingyizhou/pytorch-pose-hg-3d/issues/16 . If your pytorch version is greater than 0.1.12, you will need to disable cudnn BN by following the instruction [here](https://github.com/leoxiaobin/human-pose-estimation.pytorch).

**Comment by liuyangwen** [comment]
Hi Xingyi,
Thanks for reply. Actually, I already noticed the known issue before and I also set torch.backends.cudnn.enabled = False to disable cudnn BN, but it didn't work. Moreover, the log I submit seem to be unnormal, may you release part of the your training log for me to debug？Thanks again!

**Comment by xingyizhou** [comment]
Hi, 
  I don't have the log with me on my current machine. As I remembered, training MPJPE goes down very fast, and validation goes down slower but drops a lot after decreasing learning rate. ACC should be always > 0.9. I will suggest switching to pytorch 0.1.12 for a safe option to reproduce the result.
